D3D12Bundles/Shaders.cs

```diff
@@ -7,12 +7,19 @@ public struct VSInput
     {
         [PositionSemantic]
         public Vector3 Position;
-        //[NormalSemantic]
-        //public Vector3 Normal;
+        [NormalSemantic]
+        public Vector3 Normal;
+        [TextureCoordinateSemantic(0)]
+        public Vector2 UV;
+        [TangentSemantic]
+        public Vector3 Tangent;
+    }
```

Comment:
Reverted this to match the original HLSL shaders.  The Mutiny version is separated out as `MutinyVSInput` below.